docs/home/changelog/hotfixes-2-13.md

```diff
@@ -1,19 +1,41 @@
 ---
 title: Changelog Version 2.13
-summary: This article lists the fixed and known issues in the patch versions as well as additional new features that were added in each patch verion.
+summary: This article lists the fixed and known issues in the patch versions as well as additional new features that were added in each patch version.
 authors:
     - Jason Novich
 date: 2024-Jan-22
 ---
 
 The following is a list of the known and fixed issues for Run:ai V2.13.
 
+## Version 2.13.43 - February 15, 2024
+
+| Internal ID | Description  |
+| ---------------------------- | ---- |
+| RUN-14946 | Fixed an issue where Dashboards are displaying the hidden Grafana path. |
+
 ## Version 2.13.37
 
 | Internal ID | Description  |
 | ---------------------------- | ---- |
 | RUN-13300 | Fixed an issue where projects will appear with a status of empty while waiting for the project controller to update its status. This was caused because the cluster-sync works faster than the project controller. |
 
+## Version 2.13.35 - December 19, 2023
+
+### Release content
+
+* <!-- RUN-14441 Add ability to set node affinity for Prometheus -->Added the ability to set node affinity for Prometheus.
+
+### Fixed issues
+
+| Internal ID | Description  |
+| ---------------------------- | ---- |
+| RUN-14472 | Fixed an issue where template updates were not being applieed to the workload. |
+| RUN-14434 | Fixed an issue where `runai_allocated_gpu_count_per_gpu` was multipled by seven. |
+| RUN-13956 | Fixed an issue where changing an exisitng template created a `Promise error` on existing job templates. |
+| RUN-13825 | Fixed an issue where GPU sharing configmaps are leaked. |
```

Comment:
Very frightening and not relevant, plase remove this bug

Reply:
I don't know what "GPU sharing" means in this context. If swap - not interesting. If fractions, we can phrase that more gently as "when deleting a job that is allocated a fraction of a GPU, an associated configmap is not deleted"

Reply:
@omerbenedictrunai 
```suggestion
| RUN-13825 | Fixed an issue when deleting a job that is allocated a fraction of a GPU, an associated configmap is not deleted. |
```